src/main/java/ch/uzh/ifi/hase/soprafs24/service/PlayerService.java

```diff
@@ -29,11 +32,23 @@ public Player createPlayer(String playerName, String sessionId) {
         playerRepository.flush();
         return newPlayer;
     }
+
+    public void deletePlayer(String playerId) {
+        String cleanedPlayerId = cleanId(playerId);
+        playerRepository.deleteById(cleanedPlayerId);
+    }
+
     public Player getPlayerById(String playerId) {
-        return playerRepository.findById(playerId);
+        String cleanedPlayerId = cleanId(playerId);
+        return playerRepository.findById(cleanedPlayerId);
     }
 
     public List<Player> getPlayersInSession(String sessionId) {
-        return playerRepository.findAllBySessionId(sessionId);
+        String cleanedSessionId = cleanId(sessionId);
+        return playerRepository.findAllBySessionId(cleanedSessionId);
+    }
+
+    private String cleanId(String id) {
+        return id.replaceAll("^\"|\"$", "");
     }
```

Comment:
Could you please explain what cleanId is used for.

Reply:
Do you remeber when the ping didn't work on monday? this solves it... At least in postman (the endpoint tester I use) if i send ping request without cleaning/triming the id an error is thrown but like this it works. and for M1 (with the token) it worked for me aswell (also for client server interaction).

Reply:
I also just tried it with the client and the left most button (exit i presume) works now.

Reply:
Perfect thanks... in my second issue i will go over all holes which exist currently in our system and patch them. I may be coming back to you, but for now this LGTM.